load/UpdateDBPTextFilesets.py

```diff
@@ -61,32 +61,45 @@ def validateFileset(self, subTypeCode, bibleId, filesetId, languageRecord, lptsI
 			fullFilesetPath,
 			self.config.directory_accepted,
 			databaseName, iso3, iso1, direction]
+		print("================= Invoke BiblePublisher ========================")
+		print("cmd: ", cmd)
+		print("=========================================")
 		response = subprocess.run(cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE, timeout=120)
 		if response == None or response.returncode != 0:
 			return((Log.EROR, "BiblePublisher: " + str(response.stderr.decode("utf-8"))))
 		print("BiblePublisher:", str(response.stdout.decode("utf-8")))	
 
-		if subTypeCode == "text_json":
-			# invoke sofria
-			# this is the invocation of the sofria-cli. It should be similar to the invocation of BiblePublisher
-			cmd = [self.config.node_exe,
-				self.config.sofria_client_js,
-				fullFilesetPath,
-				self.config.directory_accepted,
-				filesetLptsName]
-			response = subprocess.run(cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE, timeout=900)
-			if response == None or response.returncode != 0:
-				return((Log.EROR, "Sofria: " + str(response.stderr.decode("utf-8"))))
-			print("Sofria:", str(response.stdout.decode("utf-8")))
-
 		return None
 
+	def invokeSofriaCli(self, fullFilesetPath, filesetLptsName):
+		# invoke sofria
+		# this is the invocation of the sofria-cli. It should be similar to the invocation of BiblePublisher
+		cmd = [self.config.node_exe,
+			self.config.sofria_client_js,
+			fullFilesetPath,
+			self.config.directory_accepted,
+			filesetLptsName]
+		print("================= Invoke Sofria Cli ========================")
+		print("cmd: ", cmd)
+		print("=========================================")
+		response = subprocess.run(cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE, timeout=900)
+		if response == None or response.returncode != 0:
+			return((Log.EROR, "Sofria: " + str(response.stderr.decode("utf-8"))))
+		print("Sofria:", str(response.stdout.decode("utf-8")))
```

Comment:
change the string to start with "Sofria invocation successful"

Reply:
@bradflood , it has been updated.